tests/irepela/homework_7/test_sorting.py

```diff
@@ -0,0 +1,11 @@
+import random
+from irepela.homework_7.sorting import bubble_sort, gnome_sort, insert_sort, select_sort
+
+
+rand_range = tuple(random.random() for _ in range(1000))
```

Comment:
move it to function, since you ocupy memory for nothing
in test it will be releases as soon as test finishes